tracing-journald/src/lib.rs

```diff
@@ -121,6 +127,14 @@ impl Subscriber {
         self
     }
 
+    /// Sets the syslog identifier for this logger.
```

Comment:
very minor nit: it might be nice to have a link to the syslog documentation defining the identifier and how it's used? not a blocker though.

Reply:
Will do.

Reply:
I've added a small paragraph about the syslog identifier.  Let me know if anything's unclear or missing :slightly_smiling_face: